cli/args/mod.rs

```diff
@@ -963,6 +963,9 @@ impl CliOptions {
     match self.sub_command() {
       DenoSubcommand::Cache(_) => GraphKind::All,
       DenoSubcommand::Check(_) => GraphKind::TypesOnly,
+      DenoSubcommand::Install(InstallFlags {
+        kind: InstallKind::Local(_),
+      }) => GraphKind::All,
```

Comment:
In the future, it would be nice to have an install command flag that doesn't install dev related stuff.

Reply:
Yeah I think we could roll that into #26121